onadata/apps/logger/models/data_view.py

```diff
@@ -169,6 +164,17 @@ def has_instance(self, instance):
 
         return True if records else False
 
+    def soft_delete(self):
+        """
+        Soft deletes a dataview by adding a deleted_at timestamp and renaming
+        the dataview by adding a deleted-at and timestamp to the name.
```

Comment:
this comment needs to explain why the `deleted-at` suffix is important, like the `XForm` `soft_delete` function's docstring does